src/layouts/BaseLayout.tsx

```diff
@@ -51,7 +51,7 @@ export const BaseLayout = ({
        * layout on initial load.
        */}
       <SkipLink />
-      <div className="mx-auto max-w-screen-2xl">
+      <div className="mx-auto max-w-screen-2xl" id="top">
```

Comment:
Want to be a little careful here since this same `id` is already being used on some pages, such as the Docs pages, and this would result in them being duplicated which isn't valid HTML.. I think we can do this another way.